projects/workflows-creator/src/lib/builder/group/group.component.html

```diff
@@ -292,14 +296,14 @@
             class="close-btn"
             (click)="
               callback(
-                getLibraryValue({target: {value: dateTime}}, inputType.DateTime)
+                getLibraryValue(nodeWithInput.node,{target: {value: dateTime}}, inputType.DateTime)
```

Comment:
Seems like code is not formatted properly @yeshamavani 
Not sure how formatting is done in this project, you might wanna look @yeshamavani